services/dynamic/dynamic-json.service.js

```diff
@@ -1,62 +1,19 @@
 'use strict'
 
-const Joi = require('@hapi/joi')
-const jp = require('jsonpath')
-const { renderDynamicBadge, errorMessages } = require('../dynamic-common')
 const { createRoute } = require('./dynamic-helpers')
-const { BaseJsonService, InvalidParameter, InvalidResponse } = require('..')
-
-module.exports = class DynamicJson extends BaseJsonService {
-  static get category() {
-    return 'dynamic'
-  }
+const jsonPath = require('./json-path')
+const { BaseJsonService } = require('..')
 
+module.exports = class DynamicJson extends jsonPath(BaseJsonService) {
```

Comment:
Admittedly just another personal stylistic preference, but this feels like a case where favoring composition over inheritance might be a bit more readable/grokable